core/controllers/base.py

```diff
@@ -357,8 +356,12 @@ def validate_and_normalize_args(self):
         """
         handler_class_name = self.__class__.__name__
         request_method = self.request.environ['REQUEST_METHOD']
+
+        # For HEAD requests, the schema of GET handler is validated
+        # against request arguments
```

Comment:
```suggestion
        # For HEAD requests, we use the schema of GET handler, 
        # because HEAD returns just the handlers of the GET request.
```

Reply:
done